scripts/kotlin_generator.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python3
+import json
+import argparse
+import os
+
+parser = argparse.ArgumentParser(description='Create Kotlin file from Json schema')
+parser.add_argument('-s', '--source', dest='json_schema',
+                   help='Json shema file', required=True)
+
+args = parser.parse_args()
+
+# Capitalize also change the next letter.
+def firstLetterUp(str):
```

Comment:
should be `first_letter_up` as per https://www.python.org/dev/peps/pep-0008/#id43

(maybe running the file through a pep8 checker like pylint would be good)

Reply:
done